plugins/sentry/config/initializers/sentry.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+if dsn = ENV['SENTRY_DSN']
+  Sentry.init do |config|
+    config.dsn = dsn
+    config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+    config.environment = Rails.env
+
+    # We recommend adjusting this value in production:
+    config.traces_sample_rate = 0.5
```

Comment:
prefer 0 (at least for the initial rollout), we had issues with this in other apps